apps/strapi-cms/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": [
     "@strapi/typescript-utils/tsconfigs/server",
-    "../../tsconfig.json"
```

Comment:
I would bring back the configurations that are no longer there

Reply:
As you asked i reverted to previus configuration this file. I am not sure about it because this re-introduce the issue that i had talk in [this comment](https://github.com/pagopa/developer-portal/pull/592#discussion_r1471528881) as you can see in this video:
[Screencast from 01-02-2024 16:06:12.webm](https://github.com/pagopa/developer-portal/assets/24475319/32d815f8-3940-4fc1-bad6-d91809954c9b)

It is not something that prevent to update permission, because if user reload page data seems updating correctly, but it is quite annoying ! It slow down the page and return error both i console and strapi snakbar.

Reply:
we have seen that the problem is solved by overriding the `declaration` option on `tsconfig.json` file.

I would make the following changes:
```
...
"compilerOptions": {
    // with declaration to true strapi raise an runtime error
    // error: Cannot read properties of undefined (reading 'type')
    // TypeError: Cannot read properties of undefined (reading 'type')
    // at /node_modules/@strapi/plugin-users-permissions/server/services/users-permissions.js:115:39
    "declaration": false,
    "outDir": "dist",
    ...
```